CHANGELOG.rst

```diff
@@ -2,6 +2,36 @@
 SDK Changelog
 =============
 
+3.3.0-alpha.2 (2025-08-?)
+==========================
+
+Third preview release of the upcoming 3.3.0 version.
+
+.. note:: This release adds the ability to opt-in to the new experimental form renderer.
+   The feature can be enabled on a per-form basis through the admin interface.
+
+New features
+------------
+
+* [open-formulieren/open-forms#5575] Implemented WMS layers in the map component.
```

Comment:
Should it be like this or just `[#5575]`? I don't remember the correct one but I see all the previous ones have the short one. Of course it's not clickable but just curious.

Reply:
The correct way to refer to the backend issue (because it is an issue created there) is indeed this one but I mentioned this because I saw in the code that we do not do it this way.

Reply:
indeed - the people reading the changelogs are not expected to understand this format so we just mention the issue numbers. Possibly we have a future improvement here to properly link to SDK/backend issues, but for now this is good enough.